components/brave_shields/resources/panel/components/advanced-controls-content/style.ts

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components'
+import globalIconUrl from '../../assets/globe.svg'
+import listsIconUrl from '../../assets/lists.svg'
+
+export const SettingsBox = styled.div`
+  padding: 17px var(--pad-x);
+
+  a {
+    font-size: 13px;
+    font-weight: 600;
+    text-decoration: none;
+    display: flex;
+    align-items: center;
+    margin-bottom: 20px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+`
+
+export const SettingsTitle = styled.div`
+  color: ${(p) => p.theme.color.text02};
+  font-family: Poppins;
+  font-size: 12px;
+  letter-spacing: 0.01em;
+  margin-bottom: 14px;
+`
+
+export const SettingsDesc = styled(SettingsTitle)`
+  color: ${(p) => p.theme.color.text03};
+`
+
+export const GlobeIcon = styled.i`
+  display: block;
+  width: 17px;
+  height: 17px;
+  background-image: url(${globalIconUrl});
```

Comment:
how about inline svg? Looks like links are mean to change color on hover anyway.

Reply:
Can we do this as a follow up? Would be more elegant after adding the webpack plugin. SVGs tend to be long, inlining can pollute the file.

Reply:
Yes I meant inline via a separate component like we do for brave-ui icons, or otherwise a separate component. Follow-up is ok I suppose, but can also just use [svg2jsx](https://svg2jsx.com/) and then follow-up with a plugin if you like.